tests/tests_pytorch/loops/test_evaluation_loop_flow.py

```diff
@@ -74,63 +71,6 @@ def backward(self, loss):
     assert opt_closure_result.item() == 171
 
 
-def test__eval_step__eval_step_end__flow(tmpdir):
-    """Tests that only training_step can be used."""
-
-    class TestModel(DeterministicModel):
-        def training_step(self, batch, batch_idx):
-            acc = self.step(batch, batch_idx)
-            acc = acc + batch_idx
-            self.training_step_called = True
-            return acc
-
-        def validation_step(self, batch, batch_idx):
-            self.validation_step_called = True
-            if batch_idx == 0:
-                out = ["1", 2, torch.tensor(2)]
-            if batch_idx > 0:
-                out = {"something": "random"}
-            self.last_out = out
-            return out
-
-        def validation_step_end(self, out):
-            self.validation_step_end_called = True
-            assert self.last_out == out
-            return out
```

Comment:
Since this does not apply anymore, the test would be come equivalent to the one above. So removing it.